cypher/cypher-docs/src/docs/dev/execution-plans.asciidoc

```diff
@@ -3,34 +3,39 @@
 
 [abstract]
 --
-This section describes the characteristics of a query's execution plan, along with details about each of the operators.
+This section describes the characteristics of query execution plans and provides details about each of the operators.
 --
 
 * <<execution-plan-introduction, Introduction>>
 * <<execution-plan-operators-summary, Execution plan operators>>
-* <<execution-plans-dbhits, DbHits (Database hits)>>
+* <<execution-plans-dbhits, Database hits (DbHits)>>
 * <<query-shortestpath-planning, Shortest path planning>>
 
 
 [[execution-plan-introduction]]
 == Introduction
 
 The task of executing a query is decomposed into _operators_, each of which implements a specific piece of work.
-The operators are combined into a tree-like structure called an _execution plan_, in which each operator is represented as a node in the tree.
-Each operator takes as input zero or more rows, and produces as output zero or more rows -- the output from one operator thus forms the input for the next operator.
-Operators that join two branches will effectively combine input from two incoming streams and produce a single output.
+The operators are combined into a tree-like structure called an _execution plan_.
+Each operator in the execution plan is represented as a node in the tree.
+Each operator takes as input zero or more rows, and produces as output zero or more rows.
+The output from one operator becomes the input for the next operator.
+Operators that join two branches in the tree combines input from two incoming streams and produce a single output.
 
 .Evaluation model
-Evaluation of the execution plan begins at the _leaf_ nodes of the tree.
-Leaf nodes have no input rows and generally comprise operators such as scans and seeks which obtain the data directly from the storage engine, thus incurring <<execution-plans-dbhits, database hits>>.
-Any rows produced by leaf nodes are then piped into their parent nodes, which in turn pipe their rows to their parent nodes and so on, all the way up to the root node, which produces the final results of the query.
+Evaluation of the execution plan begins at the leaf nodes of the tree.
+Leaf nodes have no input rows and generally comprise operators such as scans and seeks.
+These operators obtain the data directly from the storage engine, thus incurring <<execution-plans-dbhits, database hits>>.
```

Comment:
It is a single paragraph. It might look like there's an empty line in the diff because the changed line is so long it needs to be soft-broken to display it.